app/frontend/stylesheets/application.tailwind.css

```
@@ -32,7 +32,7 @@
   }
 }
 
-@plugin "daisyui" {
+@plugin "daisyui/index.js" {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)